js/ide.js

```diff
@@ -7,7 +7,7 @@ import CodeMirror from 'codemirror/lib/codemirror.js';
 import moment from 'moment';
 import tokml from 'tokml';
 import togpx from 'togpx';
-import * as _saveAs from 'file-saver';
+import {saveAs} from 'file-saver';
```

Comment:
The best introduction into ES6 modules I've found is [the chapter in Exploring ES6](http://exploringjs.com/es6/ch_modules.html).